<?xml version="1.0" encoding="UTF-8"?>
<launch>        
    <node pkg="depth_noise" name="depth_noise" type="depth_noise.py"  output="screen">        
        <param name="in_topic" value="/camera/depth/image_raw" />
        <!--<param name="std" value="0.02" /> -->
        <param name="std" value="0.03" />
    </node>
    
    <node pkg="depth_noise" name="image_converter" type="image_converter.py"  output="screen">        
        <param name="in_topic" value="/camera/rgb/image_raw"/>
        <param name="out_type" value="rgb8"/>
    </node>

</launch>
